package.json

```diff
@@ -26,7 +26,10 @@
     "htmlcov:darwin": "open coverage/index.html",
     "clean": "rm -rf coverage/ dist/ .nyc_output/"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@types/chance": "^1.0.4",
```

Comment:
are these dependencies or dev dependencies?

Reply:
This must be moved to be a dev dep... or just removed.  The tests don't need chance for what you're doing